guides/v2.2/install-gde/prereq/nginx.md

```diff
@@ -218,6 +218,9 @@ Magento requires several [PHP extensions]({{ page.baseurl }}/install-gde/prereq/
 1. Install `php-fpm`:
 
 		yum -y install php70w-fpm
+   
+   If you are on Centos 7.1 and want to install PHP7.1-FPM, do:
```

Comment:
Remove ", do" before the colon.

Reply:
Please also make this change in the v2.3 version of this file.

Reply:
@shrielenee 

k will update this sometimes today

Reply:
@shrielenee 

Updated please accept and merge this. I will update the v2.3 later on.

Reply:
I'll wait until you add the v2.3 changes to this PR. Thanks in advance!

Reply:
@shrielenee 

Updated for v2.3 and also amend on nginx user and group in both PRs

Reply:
Hey @asrar7787 - We still need you to update the 2.3 version. If you do that and push changes I'll merge this! Let me know.